packages/web/components/one-click-trading/intro-one-click-svg.tsx

```diff
@@ -0,0 +1,359 @@
+export const IntroSvg = () => {
```

Comment:
Why do we need a jsx component for the SVG? We need some runtime customization? 
Can't we just use it like a next/image and import it like an image?